src/lib/components/sidebar.svelte

```diff
@@ -4,26 +4,26 @@
 	import { onMount } from 'svelte';
 	let successCriteriaMap = {};
 	let criteriaPerPrincipe = {};
+	let baseUrl = `/${urlData.url.website.slug}/${urlData.url.slug}`;
+
+	console.log(urlData);
+	const criteriaSlice = urlData.url.checks.flatMap((check) =>
+		check.succescriteria.map((criteria) => criteria.index)
+	);
+
+	criteriaSlice.forEach((index) => {
```

Comment:
Dit heb jij niet geschreven volgens mij maar kan je wel hierbij zetten in een comment wat deze loop doet?

Reply:
Ga ik doen, ik heb ook een hele aparte taak aangemaakt om comments bij alle JS code toe te voegen.